### PR TITLE
Warn to check user faction chat when chain is near to bonus hit

### LINKED ISSUE #410: Warning : Check faction chat when chain is close to bonus hit
Warning on the attack loader if an active chain is close to a bonus hit. Warning would advice the player to check the faction chat.

Suggeted by _MajorWhiteSnow [2485676]_.

### PR DESCRIPTION
**Pull request by Sashank999**
Closes #410 .

### CODE REVIEW

File: torntools/views/settings/settings.html [hunk 1]
```diff
@@ -349,6 +349,7 @@ <h2>Preferences</h2>
 							<div class="option" id="profile-show_id"><input type="checkbox" />Reformat Profile page headings as "USERNAME [ID]"</div>
 							<div class="option" id="profile-status_indicator"><input type="checkbox" />Show user's status indicator next to their name</div>
 							<div class="option" id="profile-notes"><input type="checkbox" />Keep notes on profiles</div>
+							<div class="option" id="profile-show_chain_warning"><input type="checkbox" />Warn to check user faction chat when chain is near to bonus hit</div>
```
Comment (DeKleineKobini) [inline torntools/views/settings/settings.html:352]: New tag

File: torntools/scripts/js/globalFunctions.js [hunk 2]
```diff
@@ -1489,6 +1489,7 @@ const STORAGE = {
 				status_indicator: true,
 				block_ally_attacks: false,
 				notes: true,
+				show_chain_warning: true,
```
Comment (DeKleineKobini) [inline torntools/scripts/js/globalFunctions.js:1492]: Not sure if this should be on by default. I'll let @Mephiles decide.

**Review by DeKleineKobini — CHANGES_REQUESTED** [review]

File: torntools/scripts/js/globalFunctions.js (hunk 2 above)
Comment (Mephiles) [reply torntools/scripts/js/globalFunctions.js:1492]: Nah, let's put it default: false.

File: torntools/views/settings/settings.html (hunk 1 above)
Comment (Sashank999) [reply torntools/views/settings/settings.html]: Done.

File: torntools/scripts/js/globalFunctions.js (hunk 2 above)
Comment (Sashank999) [reply torntools/scripts/js/globalFunctions.js]: Done.

**Review by DeKleineKobini — APPROVED** [review]

**Pull request merged**